crypto/fipsmodule/bn/bytes.c

```diff
@@ -212,6 +212,9 @@ void bn_assert_fits_in_bytes(const BIGNUM *bn, size_t num) {
   size_t tot_bytes = bn->width * sizeof(BN_ULONG);
   if (tot_bytes > num) {
     CONSTTIME_DECLASSIFY(bytes + num, tot_bytes - num);
+// Avoids compiler error: unused variable 'byte' or 'word'
+// The assert statements below are only effective in DEBUG builds
```

Comment:
Huh, why doesn't this hit the little endian path for release mode?

Reply:
Only in release mode (when the assert statements are no-ops) do the values go unused. I didn't catch this due to only running debug builds locally.